rholang/src/main/scala/coop/rchain/rholang/interpreter/RhoType.scala

```diff
@@ -60,4 +61,13 @@ object RhoType {
 
     def apply(s: String): Par = GUri(s)
   }
+
+  object GDeployerId {
```

Comment:
In order to be consistent with the other RhoType names this should be called `DeployerId`